src/server/svr_func.c

```diff
@@ -7193,3 +7193,38 @@ action_opt_bf_fuzzy(attribute *pattr, void *pobj, int actmode)
 
 	return PBSE_NONE;
 }
+
+/**
+ * @brief
+ * 		sets the given value for an attribute of any object from server
+ *
+ * @param[in]	pattr	-	pointer to attribute being set
+ * @param[in]	pobj	-	pointer to attribute definition
+ * @param[in]	value	-	value to be set
+ *
+ * @return	void
+ *
+ * @par MT-Safe: No
+ * @par Side Effects: None
+ *
+ */
+void
+set_attr_svr(attribute *pattr, attribute_def *pdef, char *value)
+{
+	attribute tempat;
+
+	if (pattr == NULL || pdef == NULL) {
+		(void)sprintf(log_buffer,  "Invalid pointer to attribute or its definition");
```

Comment:
use snprintf() (and other uses in this function)

Reply:
Done